### DISCUSSION TITLE
Translation: from the Event Conditions and Actions from the chatwoot automation modal

### DISCUSSION THREAD

**Discussion by paulovitorba** [chatwoot/chatwoot#7651] [Translations]
Hello chatwoot community friends!
I've been looking for a while where I can translate the selection fields for the Events in the automation modal as I marked in the image, can someone show me the correct path where I can translate these options?

Translation: from the Event Conditions and Actions from the chatwoot automation modal
link print automation modal:
https://prnt.sc/sKcCXY_7-kjd
https://prnt.sc/B0lo1POozI7w
https://prnt.sc/7UUhhuDhFvgx

**Comment by jtayrone** [comment]
Certainly, here is the corrected English version:

Regarding translations, you can follow and contribute at [translation guide](https://www.chatwoot.com/docs/contributing/translating-chatwoot-to-your-language) for contributing to Chatwoot.

But if you want to make changes, follow the path of the options you mentioned:

[app/javascript/dashboard/routes/dashboard/settings/automation/constants.js
](https://github.com/chatwoot/chatwoot/blob/10d6e9551df302dd90bef9b83bf4299e4693b775/app/javascript/dashboard/routes/dashboard/settings/automation/constants.js)
Remember to recompile your installation at the end:

```bash
sudo -i -u chatwoot
cd chatwoot
rake assets:precompile RAILS_ENV=production
exit
systemctl daemon-reload && systemctl restart chatwoot.target
```

I hope I was helpful! 😎👍

**Reply by paulovitorba** [reply]
Thank you it worked perfectly!

**Comment by casperbj2612** [comment]
@jtayrone Hi! I am new in using chatwoot, and I am trying out the self hosted version. I am facing a bunch of troubles, and hope to gain an advice or two.. I want to integrate a language detection software, to enable the function within automation to detect the language within an email. Do you have any advice for that? Thank you in advance!
